matrix/matrix.yaml

```diff
@@ -9,21 +9,5 @@
       "periodic": 5
       "do":
         "task": matrix.tasks.health
+        "gating": true
```

Comment:
From our IRC convo:

@johnsca for the gating stuff, we actually want health and matrix not to gate by default, even though "gating" defaults to "True" at a higher level, which is why I put the gating check in "task".
13:52 cory_fu: we can discuss it after the break ... we might need to do a more extensive rethink of our logic.
13:53 (Or just add "gating: false" to a lot of our tests, rather than "gating: true" to some of them ...)